docs/tutorials/getting_started.md

```diff
@@ -395,57 +412,35 @@ To restart the CU Pod execute:
 `microk8s.kubectl -n ran delete pod cu-0`
 ```
 
-## 4. Deploy Charmed OAI RAN DU
+After adding the network configuration the CU and the DU should change their state to `active/idle`. 
```

Comment:
We need to remove the lines between 408-413, as this is not necessary any more.

Reply:
The problem is that they are. I'll talk about it in the daily.
The interface itself didn't solve this issue.